core/services/relay/evm/chain_components_test.go

```diff
@@ -207,6 +207,7 @@ func TestContractReaderEventsInitValidation(t *testing.T) {
 }
 
 func TestChainComponents(t *testing.T) {
+	t.Skip("Flakey test")
```

Comment:
Do we have a ticket?
```suggestion
	testutils.SkipFlakey(t, <ticket>)
```

Reply:
Fixed